src/Fmt_ast.ml

```diff
@@ -1207,8 +886,8 @@ and fmt_pattern c ?pro ?parens ({ctx= ctx0; ast= pat} as xpat) =
 
 and fmt_fun_args c ?(pro = fmt "") args =
   let fmt_fun_arg = function
-    | Val (Nolabel, xpat, None) -> fmt_pattern c xpat
-    | Val
+    | Sugar.Val (Nolabel, xpat, None) -> fmt_pattern c xpat
+    | Sugar.Val
```

Comment:
It's not needed to prefix with `Sugar.` every time, only once

Reply:
It seems to be needed, and since it indicates where the constructor comes from I prefer this rather than opening Sugar.

Reply:
The existing code uses the style where type annotations are added to function arguments instead. See e.g. `fmt_longident`.

Reply:
I meant qualifying the first and not the other is enough for the compiler to known where the constructors come from. Type annotations is good too.